youthful-piety/modernized/05-chapter-04.adoc

```diff
@@ -698,11 +698,14 @@ He spoke to them with remarkable fervency of spirit and very solemn feelings,
 and it appeared to be a message of love to their soul a.
 
 On taking each by the hand, he spoke nearly as follows:
-"`You see me now upon my deathbed.
+
+[quote]
+____
+You see me now upon my deathbed.
```

Comment:
I would not use the [quote] tag here, because its not a quote from a book, or a sermon, or something printed or published. It's just what the kid said. So I would leave this the way it was... with just the whole thing in quotes. 👍🏻

Reply:
I offset the line above this, I thought it looked a little better in the PDF, but I will change it back if you want.